Dungeons and Dragons Tool/dndtool/lib/dndtool/dice_server.ex

```diff
@@ -0,0 +1,27 @@
+defmodule DnDTool.Dice_Server do
```

Comment:
Is there a reason this has to be a server? It seems that it doesn't have any state (that it uses)